GPy/core/gp.py

```diff
@@ -378,27 +373,28 @@ def predict_jacobian(self, Xnew, kern=None, full_cov=True):
             dK_dXnew_full[i] = kern.gradients_X(one, Xnew, self._predictive_variable[[i]])
 
         if full_cov:
-            dK2_dXdX = kern.gradients_XX(one, Xnew, cov=False)
+            dK2_dXdX = kern.gradients_XX(one, Xnew)
         else:
```

Comment:
Could you push the changes i the slicing, please? So I'll check the plots. Many thanks!